docs/conf.py

```diff
@@ -336,7 +334,7 @@ def parse_version():
 # to. Example RST source for 'py' prefix:
 #     :py:func:`platform.dist`
 intersphinx_mapping = {
-  'py': ('https://docs.python.org/3.5', None)
+  'py': ('https://docs.python.org/3', None)
```

Comment:
This change would be useful outside of this PR.

Reply:
:+1: Done in #282